src/scripts/metarpheus/index.js

```diff
@@ -33,25 +35,29 @@ function mkDirsIfNotExist(filePath) {
   return Promise.resolve();
 }
 
+const metarpheusJSConfig = getMetarpheusJSConfig(ts);
```

Comment:
I would call this `metarpheusConfig`/`getMetarpheusConfig` (general for both JS and TS)